rsconf/component/sirepo_jupyterhub.py

```diff
@@ -18,6 +18,10 @@ class T(rsconf.component.jupyterhub.T):
     def internal_build_compile(self):
         # TODO(e-carlin): should this come from somewhere or ok to hardcode?
         self.__jupyterhub_run_d = pkio.py_path('/srv/jupyterhub')
+
+        # Must set directly on hdb so it is available in
+        # rsconf.component.jupyterhub.T.internal_build_compile
+        self.hdb[self.name].sirepo_uri = f'https://{self.hdb.sirepo.vhost}'
```

Comment:
This should be set in sirepo_config, because it is the point when you know sirepo is passing its config explicitly to sirepo_jupyterhub.

Reply:
c9cbcdc